test/functional/postalcode/dummy/us.json

```diff
@@ -0,0 +1,102 @@
+{ "type": "Feature", "properties": { "ISO": "US", "Country": "United States", "Language": "EN", "ID": 2004429781, "Region1": "Alabama", "Region2": "Autauga", "Region3": "", "Region4": "", "Postcode": "36003", "Locality": "Autaugaville", "Suburb": "", "Street": "Shiloh Cr", "Range": "400 - 47236099", "Building": "", "Elevation": 49, "ISO2": "US-AL", "FIPS": "01001", "NUTS": "", "HASC": "US.AL.AU", "STAT": "", "Timezone": "America/Chicago", "UTC": "-06:00", "DST": "-05:00" }, "geometry": { "type": "Point", "coordinates": [ -86.728501, 32.488121 ] } },
+{ "type": "Feature", "properties": { "ISO": "US", "Country": "United States", "Language": "EN", "ID": 2004429782, "Region1": "Alabama", "Region2": "Autauga", "Region3": "", "Region4": "", "Postcode": "36003", "Locality": "Autaugaville", "Suburb": "", "Street": "Shiloh Drive", "Range": "499 - 3101", "Building": "", "Elevation": 49, "ISO2": "US-AL", "FIPS": "01001", "NUTS": "", "HASC": "US.AL.AU", "STAT": "", "Timezone": "America/Chicago", "UTC": "-06:00", "DST": "-05:00" }, "geometry": { "type": "Point", "coordinates": [ -86.734990, 32.488821 ] } },
+{ "type": "Feature", "properties": { "ISO": "US", "Country": "United States", "Language": "EN", "ID": 2004429783, "Region1": "Alabama", "Region2": "Autauga", "Region3": "", "Region4": "", "Postcode": "36003", "Locality": "Autaugaville", "Suburb": "", "Street": "Smutty Street", "Range": "2613 - 2617", "Building": "", "Elevation": 49, "ISO2": "US-AL", "FIPS": "01001", "NUTS": "", "HASC": "US.AL.AU", "STAT": "", "Timezone": "America/Chicago", "UTC": "-06:00", "DST": "-05:00" }, "geometry": { "type": "Point", "coordinates": [ -86.654865, 32.429960 ] } },
+{ "type": "Feature", "properties": { "ISO": "US", "Country": "United States", "Language": "EN", "ID": 2004429784, "Region1": "Alabama", "Region2": "Autauga", "Region3": "", "Region4": "", "Postcode": "36003", "Locality": "Autaugaville", "Suburb": "", "Street": "S Pickett Street", "Range": "81 - 320", "Building": "", "Elevation": 49, "ISO2": "US-AL", "FIPS": "01001", "NUTS": "", "HASC": "US.AL.AU", "STAT": "", "Timezone": "America/Chicago", "UTC": "-06:00", "DST": "-05:00" }, "geometry": { "type": "Point", "coordinates": [ -86.650612, 32.430410 ] } },
+{ "type": "Feature", "properties": { "ISO": "US", "Country": "United States", "Language": "EN", "ID": 2004429785, "Region1": "Alabama", "Region2": "Autauga", "Region3": "", "Region4": "", "Postcode": "36003", "Locality": "Autaugaville", "Suburb": "", "Street": "State Route 14", "Range": "2179 - 3699", "Building": "", "Elevation": 49, "ISO2": "US-AL", "FIPS": "01001", "NUTS": "", "HASC": "US.AL.AU", "STAT": "", "Timezone": "America/Chicago", "UTC": "-06:00", "DST": "-05:00" }, "geometry": { "type": "Point", "coordinates": [ -86.701424, 32.444020 ] } }
+] }
```

Comment:
Can we move these files into ./test/fixtures/postalcode?

@jzaefferer, given we are testing postalcodes intensively this way, why are we testing US and KR postalcodes only? Can we test them all?

How are we supposed to update this files?